Syntaxes/Chapel.tmLanguage

```diff
@@ -258,9 +269,9 @@
 		<dict>
 			<key>match</key>
 			<string>(?x)\b(
-              	abs | max | min | sqrt | write | writeln | read | readln | open | close | 
+              	abs | max | min | sqrt | write | writeln | read | readln | open | close |
 				exit
-			
+
```

Comment:
Are these changes necessary for functionality?

Reply:
No, This is not important .  I have not intended to make any changes in this portion of code .

Reply:
You may want to make a new commit undoing it. Your commit will automatically show up.

Reply:
@LouisJenkinsCS  Okay sir